HW3.postman_collection.json

```diff
@@ -0,0 +1,133 @@
+{
```

Comment:
We could add a test to check the status code in the pre-request section of that collection and that test will be used for each query of the collection.